docs/main/prerequisites.md

```diff
@@ -82,9 +82,23 @@ Due to the complexity of different ways of setting an IP address across differen
 | lighttpd            | 80  (HTTP)   | TCP      | If you have another Web server already running, such as Apache, Pi-hole's Web server will not work. You can either disable the other Web server or change the port on which `lighttpd` listens, which allows you keep both Web servers running. |
 | pihole-FTL          | 4711    | TCP      | FTL is our API engine and uses port 4711 on the localhost interface. This port should not be accessible from any other interface.|
 
-!!! info
-    The use of lighttpd on port _80_ is optional if you decide not to install the Web dashboard during installation.
-    The use of pihole-FTL on ports _67_ or _547_ is optional, but required if you use the DHCP functions of Pi-hole.
+<!-- markdownlint-disable code-block-style -->
+!!! info "Optional ports"
+    The use of lighttpd on port *80* is optional if you decide not to install the Web dashboard during installation.
+    The use of pihole-FTL on ports *67* or *547* is optional, but required if you use the DHCP functions of Pi-hole.
+
+!!! warning "Firewall configuration"
+    ***Never*** open the ports listed above to the Internet as this will lead to creating an [open resolver](http://openresolverproject.org/). Open resolvers are a severe danger both for you and other users of the Internet. There is a lot that can go wrong and could make you becoming part of a [DNS amplification attack](https://www.cloudflare.com/learning/ddos/dns-amplification-ddos-attack/) very easily. When you need to use your Pi-hole from the Internet (whilst travelling, etc.), you **must** set up a VPN to do so (see, e.g., [our Wireguard guide](../guides/vpn/wireguard/overview.md) on how to do this). This will give you safe and secure access for your Pi-hole from everywhere. This cannot be stressed enough.
```

Comment:
The link does not work.

```suggestion
    ***Never*** open the ports listed above to the Internet as this will lead to creating an open resolver. Open resolvers are a severe danger both for you and other users of the Internet. There is a lot that can go wrong and could make you becoming part of a [DNS amplification attack](https://www.cloudflare.com/learning/ddos/dns-amplification-ddos-attack/) very easily. When you need to use your Pi-hole from the Internet (whilst travelling, etc.), you **must** set up a VPN to do so (see, e.g., [our Wireguard guide](../guides/vpn/wireguard/overview.md) on how to do this). This will give you safe and secure access for your Pi-hole from everywhere. This cannot be stressed enough.
```

Reply:
Is there any other link to be used as a replacement?

Reply:
I think it is enough to have the link to cloudflare's amplification attack. They even have a small section about open resolvers.